docs/contributing.rst

```diff
@@ -67,6 +67,8 @@ The following tools are there to help you:
 
 Integration
 ~~~~~~~~~~~
+Mac OS X users: Run `./tests/mac-integration.sh` to configure the
```

Comment:
Shouldn't that be `sudo ./tests/mac-integration.sh`? For example, doesn't `mysql.server start` require any root privs?